h2/src/test/org/h2/test/scripts/testScript.sql

```diff
@@ -4424,6 +4424,9 @@ SELECT * FROM TEST2COL WHERE A=0 AND B=0;
 EXPLAIN SELECT * FROM TEST2COL WHERE A=0 AND B=0;
 >> SELECT "PUBLIC"."TEST2COL"."A", "PUBLIC"."TEST2COL"."B", "PUBLIC"."TEST2COL"."C" FROM "PUBLIC"."TEST2COL" /* PUBLIC.PRIMARY_KEY_E: A = 0 AND B = 0 */ WHERE ("A" = 0) AND ("B" = 0)
 
+EXPLAIN SELECT * FROM TEST2COL WHERE (A, B)=(0, 0);
+>> SELECT "PUBLIC"."TEST2COL"."A", "PUBLIC"."TEST2COL"."B", "PUBLIC"."TEST2COL"."C" FROM "PUBLIC"."TEST2COL" /* PUBLIC.PRIMARY_KEY_E: A = 0 AND B = 0 */ WHERE ROW ("A", "B") = ROW (0, 0)
+
```

Comment:
Please, don't add any new tests to this legacy script. You can add them to `indexes.sql` instead.

I think we need more tests including multi-column joins with other tables with both `EXPLAIN` and actual test of query results.

Reply:
Ok, I deleted the new lines.

I created a new test class: TestCompoundIndexSearch